frontend/src/pages/modelRegistry/screens/ModelDetailsView.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { Button, ClipboardCopy, DescriptionList, Flex, FlexItem } from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { RegisteredModel } from '~/concepts/modelRegistry/types';
+import DashboardDescriptionListGroup from '~/components/DashboardDescriptionListGroup';
+import EditableTextDescriptionListGroup from '~/components/EditableTextDescriptionListGroup';
+import EditableLabelsDescriptionListGroup from '~/components/EditableLabelsDescriptionListGroup';
+import RegisteredModelOwner from './RegisteredModelOwner';
+import ModelTimestamp from './ModelTimestamp';
+import { getLabels } from './utils';
+
+type ModelDetailsViewProps = {
+  registeredModel: RegisteredModel;
+};
+
+const ModelDetailsView: React.FC<ModelDetailsViewProps> = ({ registeredModel: rm }) => {
+  // eslint-disable-next-line no-console
+  console.log({ rm });
```

Comment:
I would assume we don't wanna console log this at the end right?

Reply:
True, I can remove that. Although without any code outside the JSX here the linter wants this to be an inline arrow-function expression, which means when we later consume the context to do API calls that diff will change a bunch of whitespace in here and cause conflicts if multiple people are working on this file. Probably not an issue, will remove.